README.md

```diff
@@ -10,6 +10,40 @@ Description
+* **Liveblog**: This module helps to cover events as they happen, like sport events or elections.
+* **Schedule your content**: You can plan when texts should be published, and you can also set a date to remove the article from your website. Also, you can set an expire date for images. After this, the picture will be replaced with a placeholder.
+* **Improved media handling**: With the help of the media browser, it’s very easy to add pictures, galleries, or videos to your article. You can upload new pictures to the media browser by just dragging and dropping. The pictures are cropped automatically and with the focal point, you can define the important part of the picture, which should be visible in every layout.
+* **Keep your users engaged with Riddle**: With [Riddle] (https://www.riddle.com/), any editor can quickly create professional quizzes, polls, or personality tests. Publishers can use these to generate more traffic on their sites and to monetize this traffic with lead generation or selling sponsored quiz slots. 
+* **Comments**: Use Drupal’s comment function and let users discuss about your articles.
+* **Mobile Friendly**: With the Thunder installation, you get a responsive theme for frontend and backend. When writing an article, you can check how your text will look on different smartphones and tablets with the help of the device preview.
+* **Multi-device video player**: With the easy-to-embed media player [nexxPLAY from nexx.tv] (http://www.nexx.tv/thunder), you get a powerful tool for video distribution. It enables live streaming as well as video-on-demand services for all relevant platforms. Plus, nexx.tv offers solutions for monetizing videos. Alternatively, you can include videos from Youtube or Vimeo to your article, too.
+* **Single Sign-On solution**: The Harbourmaster is a multi-site, multi-device Single Sign-On solution, developed for the media industry by our industry partner Valiton. The Harbourmaster contains a Policy Server, User Directory, Authentication Gateway, and User Manger.
+* **Facebook Instant Articles**: With Thunder, editors may publish their content easily as instant articles on Facebook. This reduces loading times on the mobile web.
+* **Analytics**: Interfaces to Google Analytics and the German audience measurement organization IVW.
+* **Ad-Integration**: We integrated Google AdSense. This enables you to embed advertisements into your website and earn money without developing your own solution.
+* **Demo Content and Guided Tour**: We added demo content and a guided tour to make it easier for newcomers to understand how Thunder works. Step by step, we explain how to add an article, how to edit an article, or how to add and sort channels. 
+* **Arrange your content**: Use categories and tags to sort your articles. Plus, you can easily create lists of content, such as the most read articles, or a package about a specific topic.
+
+NOCH OFFEN
+* Feeds: ????
+* Multilingual: ???
```

Comment:
That means, that you can have a web page in multiple languages. E.G. a german and an english version.